src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-692.mdx

```diff
@@ -0,0 +1,18 @@
+---
+subject: Android agent
+releaseDate: '2023-01-11'
+version: 6.9.2
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.9.2.zip'
+---
+
+## Fixed
+
+* Fixed an issue of missing element in harvest payload, which has caused data loss during ingest in version [6.9.1](/docs/release-notes/mobile-release-notes/android-release-notes/android-691)
+* Tightened metric data validations to prevent data loss during ingest
+* Fixed a bug with the plugin fails to tag and upload 'mapping.txt' file
```

Comment:
This statement doesn't accurately describe the problem, but was more a symptom. I'd like to change it to:

"Fixed a bug that allowed direct access to the agent plugin extension's configuration from Kotlin that could result in a failure to tag and upload obfuscation maps."